ttt_csharp/GameRules.cs

```diff
@@ -0,0 +1,40 @@
+namespace ttt_csharp
+{
+    public class GameRules
+    {
+        public bool HorizontalWin(Board board, char marker)
+        {
+            return board.Spots[0].Equals(board.Spots[1]) & board.Spots[1].Equals(board.Spots[2]) & board.Spots[0].Equals(marker)
+                | board.Spots[3].Equals(board.Spots[4]) & board.Spots[4].Equals(board.Spots[5]) & board.Spots[3].Equals(marker)
+                | board.Spots[6].Equals(board.Spots[7]) & board.Spots[7].Equals(board.Spots[8]) & board.Spots[8].Equals(marker);
```

Comment:
Not a big deal, but maybe there are ways to remove some of the duplication here. The only things changing here are the numbers.